spec/module.dd

```diff
@@ -152,9 +152,9 @@ module bar;
 import foo;  // Deprecated: module foo is deprecated
 ---------
 
-$(P $(I DeprecatedAttribute) can have an optional $(ASSIGNEXPRESSION) argument to provide a
-more expressive message. The $(I AssignExpression) must evaluate at compile time to
-a string.)
+$(P A $(I DeprecatedAttribute) can have an optional $(ASSIGNEXPRESSION) argument to provide a
+more informative message. An $(I AssignExpression) must evaluate to a string at compile time.
+)
```

Comment:
I think the start of the second sentence is fine as is. i.e. keep the "The".

Reply:
True, not all assignexp needs to be CTFEable. But the change to the end of the sentence is 👍

Reply:
Using 'A' here is better English than 'The'.
So I'll leave you to change it if you insist:-)

Reply:
No, the `A` on line 155 is good. Its the `An` on line 156 that should stay a `The`. Sorry for the confusion.